src/effects/presets/effectchainpresetmanager.cpp

```diff
@@ -359,10 +389,26 @@ void EffectChainPresetManager::setQuickEffectPresetOrder(
         m_quickEffectChainPresetsSorted.append(
                 m_effectChainPresets.value(chainPresetName));
     }
+    prependEmptyPreset(m_quickEffectChainPresetsSorted);
 
     emit quickEffectChainPresetListUpdated();
 }
 
+void EffectChainPresetManager::prependEmptyPreset(
+        QList<EffectChainPresetPointer>& pPresetList) const {
```

Comment:
We try to avoid non const references. 
```suggestion
        QList<EffectChainPresetPointer>* pPresetList) const {
```

In this case you may also consider to use m_quickEffectChainPresetsSorted directly.

Reply:
oh, okay.
I just merged it into `setQuickEffectPresetOrder` because it's only called from there anyway.